src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -589,6 +585,22 @@ public WorkAllocationFields getWorkAllocationFields() {
         return workAllocationFields;
     }
 
+    @JsonIgnore
+    public SchedulingAndListingFields getSchedulingAndListingFields() {
+        if (schedulingAndListingFields == null) {
+            this.schedulingAndListingFields = new SchedulingAndListingFields();
+        }
+        return schedulingAndListingFields;
+    }
+
+    @JsonIgnore
+    public JointParty getJointParty() {
+        if (isNull(jointParty)) {
+            this.jointParty = new JointParty();
+        }
+        return jointParty;
+    }
+
```

Comment:
Why are we using isNull in one place and == null in another?  We should have consistency.